core/src/components/chip/chip.ionic.scss

```diff
@@ -9,14 +10,13 @@ $ionic-states-hover: #{rgba(#05080f, 0.16)}; // We should review how to make thi
 
 :host {
   --background: #{globals.$ionic-color-neutral-10};
-  --border-color: transparent;
   --border-radius: #{globals.$ionic-border-radius-rounded-small};
   --color: #{globals.$ionic-color-neutral-900};
   --focus-ring-color: #{$ionic-states-focus-primary};
   --focus-ring-width: #{globals.$ionic-border-size-medium};
 
   @include globals.font-smoothing;
-  @include globals.padding(globals.$ionic-space-xs, globals.$ionic-space-xxs);
+  @include globals.padding(globals.$ionic-space-xxs, globals.$ionic-space-xs);
```

Comment:
Verified on Figma that these values had to be swapped.

Reply:
Double verified 👍